main/resources/com/google/refine/grel/FunctionDescription.properties

```diff
@@ -93,7 +93,7 @@ str_split_by_lengths=Returns the array of strings obtained by splitting s into s
 str_split=Returns the array of strings obtained by splitting s by sep. The separator can be either a string or a regex pattern. If preserveTokens is true, empty segments are preserved.
 str_starts_with=Returns a boolean indicating whether s starts with sub. For example, "food".startsWith("foo") returns true, whereas "food".startsWith("bar") returns false.
 str_to_lowercase=Returns string s converted to all lowercase characters.
-str_to_title_case=Returns string s converted into titlecase: a capital letter starting each word, and the rest of the letters lowercase. For example, ''Once upon a midnight DREARY''.toTitlecase() returns the string ''Once Upon A Midnight Dreary''.
+str_to_title_case=Returns string s converted into titlecase: a capital letter starting each word, and the rest of the letters lowercase. For example, ''Once upon a midnight DREARY''.toTitlecase() returns the string ''Once Upon A Midnight Dreary''. Optionally, a string of delimiter characters used to separate words. The first character and the first non-delimiter character after a delimiter will be captialized. For example, "jean-michel jarre".toTitlecase(" -") returns "Jean Michel Jarre".
```

Comment:
```suggestion
str_to_title_case=Returns string s converted into titlecase: a capital letter starting each word, and the rest of the letters lowercase. For example, ''Once upon a midnight DREARY''.toTitlecase() returns the string ''Once Upon A Midnight Dreary''. Optionally, a string of delimiter characters used to separate words. The first character and the first non-delimiter character after a delimiter will be captialized. For example, "jean-michel jarre".toTitlecase(" -") returns "Jean-Michel Jarre".
```

Reply:
Thanks for reviewing the changes. Also, I copied the description from the User manual and I think we might have to change that as well.

https://openrefine.org/docs/manual/grelfunctions#totitlecases